app/build.gradle

```diff
@@ -36,28 +37,24 @@ android {
         compose true
     }
     composeOptions {
-        kotlinCompilerExtensionVersion compose_compiler_version
+        kotlinCompilerExtensionVersion '1.3.2'
```

Comment:
leave in compose_compiler_version if it's already defined?

Reply:
We defined it, it was not from the template. 
We don't need it anymore.